temporal-sdk/src/main/java/io/temporal/internal/common/DebugModeUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.internal.common;
+
+public class DebugModeUtils {
+  private static final boolean TEMPORAL_DEBUG_MODE = System.getenv("TEMPORAL_DEBUG") != null;
```

Comment:
I know it's annoying but you probably want to make this false if it's a "falsy" value too. Sometimes people will set to '0' or "false" and expect it to get turned off

Reply:
Yeah, makes sense actually, let me do it as a part of this PR.